src-vue/src/views/ChangelogView.vue

```diff
@@ -1,24 +1,78 @@
 <template>
-    <div class="fc__changelog__container">
-        <el-link :underline="false" icon="DocumentCopy" target="_blank"
-            href="https://github.com/R2Northstar/Northstar/releases">
-            Open release notes
-        </el-link>
+    <div style="height: calc(100% - var(--fc-menu_height))">
+        <div v-if="releases.length === 0" class="fc__changelog__container">
+            <el-progress :show-text="false" :percentage="50" :indeterminate="true" />
+        </div>
+        <el-scrollbar v-else>
```

Comment:
The scrollbar is kinda hard to see, maybe a bit less transparent would be nice ^^

![screenshot](https://user-images.githubusercontent.com/40122905/200671657-7b3521ad-aecf-48a3-b434-d355f2fb80a6.png)

Reply:
Fixed with df2fd3b7cf0bd2633105cd94c1be192c53b81014